benchutils/tests/test_metrics.py

```diff
@@ -27,19 +28,19 @@ def test_correlation_basic(self):
 
         denominator = np.sqrt(denominator_sq)
         expected_correlation = numerator / denominator
-        observed_correlation = correlation(obs_profile, exp_profile)
+        observed_correlation = correlation(obs_profile, exp_profile_orig)
         self.assertAlmostEqual(observed_correlation, expected_correlation)
 
     def test_l2_norm_basic(self):
-        input_ = pd.Series([1, 0])
-        output = pd.Series([0, 0])
+        input_ = np.array([1, 0])
```

Comment:
I think the tests are overall too simple. Maybe add one or two more cases per function.

Reply:
I agree. Future PR.